src/server.cpp

```diff
@@ -246,6 +246,10 @@ Server::Server(
 	m_async_globals_data(""),
 	m_modchannel_mgr(new ModChannelMgr())
 {
+	m_path_common = porting::path_user + DIR_DELIM "common";
```

Comment:
Isn't this even worse than writing to mod dirs because there is one shared common path for all mods (previously modsec would block other mods trying to write to your mod folder)?

Reply:
Same as `get_worldpath`, but without the world bit

Reply:
This is a separate issue you've already raised in the other line comment, there's no need to bring it up here

Reply:
What I'm bringing up here (and what I should've stressed in the above comment) is that this is inferior to writing to mod dirs in the regard that there are no security guarantees.